spec/shared_behaviors/taggable.rb

```diff
@@ -29,6 +34,11 @@ def tag_link(s)
       expect(formatted_string).to include(tag_link('what'))
       expect(formatted_string).to include(tag_link('hey'))
       expect(formatted_string).to include(tag_link('vöglein'))
+      expect(formatted_string).to include(tag_link('ինչո՞ւ'))
+      expect(formatted_string).to include(tag_link('այո՜ո'))
+      expect(formatted_string).to include(tag_link('սեւ֊սպիտակ'))
+      expect(formatted_string).to include(tag_link('գժանո՛ց'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.